usermods/audioreactive/audio_reactive.h

```diff
@@ -799,12 +803,12 @@ class AudioReactive : public Usermod {
       if (time_now - last_time > 2)  {
         last_time = time_now;
 
-        if((fabsf(sampleReal) < 2.0f) || (sampleMax < 1.0)) {
+        if ((fabsf(sampleReal) < 2.0f) || (sampleMax < 1.0f)) {
           // MIC signal is "squelched" - deliver silence
           tmpAgc = 0;
           // we need to "spin down" the intgrated error buffer
-          if (fabs(control_integrated) < 0.01)  control_integrated  = 0.0;
-          else                                  control_integrated *= 0.91;
+          if (fabs(control_integrated) < 0.01f)  control_integrated  = 0.0f;
+          else                                   control_integrated *= 0.91f;
```

Comment:
control_integrated (the PID integrator) is double intentionally, because this value is very sensitive to quantization error. 

You can leave all operations affecting this variable without `f` suffix.